From 63384881a68250d70844cc6525bc95d1aa101263 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 25 Mar 2016 13:00:48 -0700
Subject: [PATCH 696/793] sound: soc: xilinx: Fix Makefile to build as modules

Define each file as separate module. This way, the driver can be
built as a separate module.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 sound/soc/xilinx/Makefile |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/xilinx/Makefile b/sound/soc/xilinx/Makefile
index f8d764f..8da14bd 100644
--- a/sound/soc/xilinx/Makefile
+++ b/sound/soc/xilinx/Makefile
@@ -1,3 +1,3 @@
-snd-soc-xilinx-dp-objs := xilinx-dp-pcm.o xilinx-dp-codec.o xilinx-dp-card.o
-
-obj-$(CONFIG_SND_SOC_XILINX_DP) += snd-soc-xilinx-dp.o
+obj-$(CONFIG_SND_SOC_XILINX_DP) += xilinx-dp-pcm.o
+obj-$(CONFIG_SND_SOC_XILINX_DP) += xilinx-dp-codec.o
+obj-$(CONFIG_SND_SOC_XILINX_DP) += xilinx-dp-card.o
-- 
1.7.5.4

